app/build.gradle

```diff
@@ -66,19 +66,24 @@ android {
 def AAVersion = '4.2.0'
 
 dependencies {
+    compile 'com.jakewharton:butterknife:8.8.1'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.8.1'
+    // TODO: temporary added until maven repo work again
+    compile files('../achartengine-1.2.0.jar')
     compile 'com.android.support:support-v4:25.3.1'
     compile 'com.android.support:multidex:1.0.1'
 
     compile "com.google.android.gms:play-services-base:11.0.2"
     compile "com.google.android.gms:play-services-drive:11.0.2"
     compile "com.google.android.gms:play-services-plus:11.0.2"
 
-    apt "org.androidannotations:androidannotations:$AAVersion"
+    annotationProcessor "org.androidannotations:androidannotations:$AAVersion"
```

Comment:
why is it needed?

Reply:
New gradle doesn't work with android-apt